cypress/e2e/common/visual_regression.js

```diff
@@ -0,0 +1,10 @@
+import { Then } from '@badeball/cypress-cucumber-preprocessor';
+
+Then(`{string} should not be visually different`, (pageName) => {
```

Comment:
The wording of this felt a little unclear to me, maybe something like this is more fitting?
```suggestion
Then(`the page matches the {string} baseline image`, (pageName) => {
```